### PR TITLE
docs(CONTRIBUTING.md): add section about how we structure our code

### PR DESCRIPTION
**Pull request by adrianschmidt**
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -115,6 +115,35 @@ To ensure consistency and quality throughout the source code, before being accep
 **Please note**:
 We will be happy to help you out in meeting these requirements. Don't be afraid to share your code in a pull request even if it doesn't meet the requirements yet.
 
+#### Code Structure
+
+We try to follow the "Newspaper code structure", see https://kentcdodds.com/blog/newspaper-code-structure. (Yes, we are aware that the author has archived that blog post because he doesn't follow that principle anymore, but we still think it's a good fit for us.)
+
+We put all non-function properties on top, so we want to order them with public properties first, then more important private properties, then less important private properties.
```
Comment (Kiarokh) [inline CONTRIBUTING.md:122]:
```suggestion
We try to follow the "Newspaper code structure", which in short means putting all non-function properties on top; and ordering them with public properties first, then more important private properties, then less important private properties.
```

Comment (Kiarokh) [reply CONTRIBUTING.md:122]: People can always Google stuff. If we think using this is good, there is no need to bring up excuses. Let's keep it simple.

Comment (adrianschmidt) [reply CONTRIBUTING.md:122]: Ok 👍

**Review by Kiarokh — APPROVED** [review]

**Pull request merged**